lib/common.js

```diff
@@ -7,8 +7,14 @@
 //   newEventName - name of the forwarded event (optional)
 //
 const forwardEvent = (from, to, eventName, newEventName = eventName) => {
-  from.on(eventName, (...eventArgs) => {
-    to.emit(newEventName, ...eventArgs);
+  if (from.on) {
+    from.on(eventName, (...eventArgs) => {
+      to.emit(newEventName, ...eventArgs);
```

Comment:
Why do you checking `from.on` but not checking `to.emit`?
Do you trust message destination more than message source?